SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestAutoDestroyComponent.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "SpatialFunctionalTestAutoDestroyComponent.h"
+
+USpatialFunctionalTestAutoDestroyComponent::USpatialFunctionalTestAutoDestroyComponent()
+	: Super()
+{
+#if ENGINE_MINOR_VERSION <= 23
+	bReplicates = true;
+#else 
+	SetIsReplicatedByDefault(true);
+#endif
+}
+
+
```

Comment:
Ultra nit: some extra empty lines at the end of this file and SpatialFunctionalTest.cpp

Reply:
Removed extra lines.